warehouse/migrations/versions/d738a238d781_remove_pgp_signature_support.py

```diff
@@ -0,0 +1,31 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Remove PGP signature support
+
+Revision ID: d738a238d781
+Revises: c5f718cb98ac
+Create Date: 2023-05-21 14:46:11.845339
+"""
+
+from alembic import op
+
+revision = "d738a238d781"
+down_revision = "c5f718cb98ac"
+
+
+def upgrade():
+    op.drop_column("release_files", "has_signature")
```

Comment:
Note: This migration will cause failures when deployed until the old version of Warehouse is no longer serving traffic. I plan to drop it from the PR before merging, and move it to a follow up PR.